docs/Installation.md

```diff
@@ -0,0 +1,93 @@
+# Installation Guide for ClusterLink
+The ClusterLink gateway contains three main components: the control-plane, data-plane, and gwctl (more details can be found [here](../README.md#what-is-clusterlink)).
+
+ClusterLink can be deployed in a local environment (Kind) or any remote K8s cluster environment (GKE, EKS, IBM-K8S, etc.).
+
+To deploy the ClusterLink gateway in a K8s cluster, follow these steps:
+
+1. Create and deploy certificates and the ClusterLink Deployment file.
+2. Expose the ClusterLink deployment to a public IP.
+3. Optionally, create a central gwctl component to control all gateways.
+
+Before you begin, please clone and build the project according to the [instructions](../README.md#building-clustelink).
+### 1. ClusterLink Deployment and certificates.
+In this step, we build the ClusterLink certificates and deployment files.
+
+1) Create a folder (eg. TEST_DIR) for all deployments and yaml files:
+
+        export PROJECT_DIR=`git rev-parse --show-toplevel`
+        export TEST_DIR=$PROJECT_DIR/bin/test/
```

Comment:
Why example is put it on project dir?
Also, I would rename `TEST_DIR` to `DEPLOY_DIR`.
I suggest using `export TEST_DIR=~/clusterlink-deployment`